aztec/src/main/kotlin/org/wordpress/aztec/AztecText.kt

```diff
@@ -660,6 +666,36 @@ open class AztecText : AppCompatEditText, TextWatcher, UnknownHtmlSpan.OnUnknown
         }
     }
 
+    override fun onCreateInputConnection(outAttrs: EditorInfo) : InputConnection {
+        // initialize inputConnectionEditorInfo
+        if (inputConnectionEditorInfo == null) {
+            inputConnectionEditorInfo = outAttrs
+        }
+
+        // now init the InputConnection, or replace if EditorInfo contains anything different
+        if (inputConnection == null || !EditorInfoUtils.areEditorInfosTheSame(outAttrs, inputConnectionEditorInfo!!)) {
+            // we have a new InputConnection to create, save the new EditorInfo data and create it
+            // we make a copy of the parameters being received, because super.onCreateInputConnection may make changes
+            // to EditorInfo params being sent to it, and we want to preserve the same data we received in order
+            // to compare.
+            // (see https://android.googlesource.com/platform/frameworks/base/+/jb-mr0-release/core/java/android/widget/
+            // TextView.java#5404)
+            inputConnectionEditorInfo = EditorInfoUtils.copyEditorInfo(outAttrs)
+            val localInputConnection = super.onCreateInputConnection(outAttrs)
+            if (localInputConnection == null) {
+                // in case super returns null, let's just observe the base implementation, no need to make
+                // an InputConnectionWrapper of a null target
+                return localInputConnection
+            }
+            // if non null, wrap the new InputConnection around our wrapper
```

Comment:
np: maybe we can mention in the comment this is for debugging only

Reply:
oh totally - I was intending to remove this line once the PR was approved and forgot to mention it here.
Are we ok with that or would you prefer that we add an actual comment in code and then remove it altogether before merging (if it gets merged)?

Reply:
That's totally cool if we just remove it. My concern was that it might be confusing if the comment is left there, with the commented out code, and no explanation about the logging. :+1: